README.rst

```diff
@@ -47,6 +47,32 @@ Does not provide:
 - SFTP. Security is for the weak.
 - Less common commands.
 
+Monkey Patching
+---------------
+
+Sometimes you may want to call a library that uses requests with an ftp URL.
+First ensure that the library does not take a session parameter:
```

Comment:
First, check whether the library takes a session parameter. If it does, you can use the FTPSession class (or FTPAdaptor), which is the preferred approach